lib/changelog/notifier.ex

```diff
@@ -164,4 +177,20 @@ defmodule Changelog.Notifier do
       |> Mailer.deliver_later()
     end
   end
+
+  defp deliver_submitter_decline_email(person, item) do
+    if person.settings.email_on_submitted_news do
```

Comment:
I think in the case of this kind of feedback, we should send the email _regardless_ of their notification settings. This is more of a personal email than it is a notification, so we'd definitely want it to send.

Reply:
Yeah, that makes sense to me.